File: skills/astra-toolkit/clients/java/examples/admin-drop-database.java
import com.datastax.astra.client.DataAPIClient;
import com.datastax.astra.client.admin.AstraDBAdmin;
import java.util.UUID;

public class Example {
  public static void main(String[] args) {
    DataAPIClient client = new DataAPIClient(System.getenv("ASTRA_DB_APPLICATION_TOKEN"));

    AstraDBAdmin admin = client.getAdmin();

    admin.dropDatabase(UUID.fromString("**DATABASE_ID**"));
  }
}

File: skills/astra-toolkit/clients/csharp/examples/admin-drop-database.cs
using DataStax.AstraDB.DataApi;

namespace Examples;

public class Program
{
  static async Task Main()
  {
    var client = new DataAPIClient(System.Environment.GetEnvironmentVariable("ASTRA_DB_APPLICATION_TOKEN"));

    var admin = client.GetAstraDatabasesAdmin();

    await admin.DropDatabaseAsync("**DATABASE_ID**");
  }
}
